New_PlatformGame/Assets/ScaffoldingScript.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking.PlayerConnection;
+
+public class ScaffoldingScript : MonoBehaviour
+{
+    // Start is called before the first frame update
+    public GameObject coinPrefab;
+    public const int percentage = 30;
+    void Start()
+    {
+        int random = Random.Range(1, 10);
+        Debug.Log(random);
+        if (random <= 3)
+        {
+            Instantiate(coinPrefab, transform.position + new Vector3(0, 1f, 0), Quaternion.identity);
```

Comment:
```suggestion
            Instantiate(coinPrefab, transform.position + new Vector3(0, 1f, 0), Quaternion.identity);
```
여기도 벡터 부분을 상수로 때리거나, 혹은 Vector3.Up을 이용하는 방법도 있다. 0,1,0의 값을 지원하는 Vector3에 내장되어있는 enum 형태의 상수이다.